policy-utils/src/error.rs

```diff
@@ -11,8 +11,8 @@
 
 use err_derive::Error;
 #[cfg(feature = "std")]
-use rustls::{CipherSuite, Error as TLSError};
-use std::{string::String, time::SystemTimeError};
+use std::string::String;
+use std::time::SystemTimeError;
```

Comment:
When feature "std" is not enabled, `std::time::SystemTimeError` will still be `use`d. Is this what you intended?

Reply:
This is a bit messed up. I'll fix it.